packages/api/src/lib/structures/Route.ts

```diff
@@ -33,7 +33,7 @@ export abstract class Route extends Piece {
 	public constructor(context: PieceContext, options: RouteOptions = {}) {
 		super(context, options);
 
-		const api = this.context.server.options;
+		const api = this.container.server.options;
 		this.router = new RouteData(`${api.prefix ?? ''}${options.route ?? ''}`);
```

Comment:
```suggestion
		this.router = new RouteData(`${api.prefix ?? ''}${options.route ?? this.name ?? ''}`);
```

This is something I noticed in sapphire sample bot but shouldn't we try for the piece name as well? That's what we do for commands, listeners, preconditions and arguments as well.